concepticondata/conceptlists/Marchese-1983-311.tsv

```diff
@@ -0,0 +1,312 @@
+Marchese-1983-311-69	69	maïs	506	MAIZE	but	LES VEGETAUX
+Marchese-1983-311-70	70	mil	931	MILLET	mil	LES VEGETAUX
+Marchese-1983-311-71	71	palmier à huile			oil palm tree	LES VEGETAUX
+Marchese-1983-311-72	72	palmier (raphia)	1181	PALM TREE	palm (raffia)	LES VEGETAUX
+Marchese-1983-311-73	73	papaye	2445	PAPAYA	papaya	LES VEGETAUX
+Marchese-1983-311-74	74	piment	919	CHILI PEPPER	chili pepper	LES VEGETAUX
+Marchese-1983-311-75	75	riz	926	RICE	rice	LES VEGETAUX
+Marchese-1983-311-76	76	tabac	974	TOBACCO	tobacco	LES VEGETAUX
+Marchese-1983-311-77	77	taro	351	TARO	taro	LES VEGETAUX
+Marchese-1983-311-78	78	albinos			albino	L'ÊTRE HUMAIN
+Marchese-1983-311-79	79	ami	1325	FRIEND	friend	L'ÊTRE HUMAIN
+Marchese-1983-311-80	80	bété			stupid	L'ÊTRE HUMAIN
+Marchese-1983-311-81	81	blanc	1335	WHITE	white	L'ÊTRE HUMAIN
+Marchese-1983-311-82	82	enfant	2099	CHILD	child	L'ÊTRE HUMAIN
+Marchese-1983-311-83	83	épouse	1199	WIFE	wife	L'ÊTRE HUMAIN
+Marchese-1983-311-84	84	époux	1200	HUSBAND	husband	L'ÊTRE HUMAIN
+Marchese-1983-311-85	85	femme	962	WOMAN	women	L'ÊTRE HUMAIN
+Marchese-1983-311-86	86	homme (être humain)	683	PERSON	man (human)	L'ÊTRE HUMAIN
+Marchese-1983-311-87	87	homme (mâle)	1554	MAN	human (male)	L'ÊTRE HUMAIN
+Marchese-1983-311-88	88	mère	1216	MOTHER	mother	L'ÊTRE HUMAIN
+Marchese-1983-311-89	89	père	1217	FATHER	dad	L'ÊTRE HUMAIN
+Marchese-1983-311-90	90	aile	1257	WING	wing	L'ÊTRE HUMAIN
+Marchese-1983-311-91	91	barbe	678	BEARD	beard	L'ÊTRE HUMAIN
+Marchese-1983-311-92	92	bosse	1903	SWELLING	bump	L'ÊTRE HUMAIN
+Marchese-1983-311-93	93	bouche	674	MOUTH	mouth	L'ÊTRE HUMAIN
+Marchese-1983-311-94	94	boyaux	1334	GUTS	hoses	L'ÊTRE HUMAIN
+Marchese-1983-311-95	95	bras	1673	ARM	arms	L'ÊTRE HUMAIN
+Marchese-1983-311-96	96	chair	2259	FLESH	flesh	L'ÊTRE HUMAIN
+Marchese-1983-311-97	97	cheveu	2648	HAIR (HEAD)	hair	L'ÊTRE HUMAIN
+Marchese-1983-311-98	98	coeur	1223	HEART	heart	L'ÊTRE HUMAIN
+Marchese-1983-311-99	99	corne	1393	HORN (ANATOMY)	horn	L'ÊTRE HUMAIN
+Marchese-1983-311-100	100	cou	1333	NECK	neck	L'ÊTRE HUMAIN
+Marchese-1983-311-101	101	dent	1380	TOOTH	tooth	L'ÊTRE HUMAIN
+Marchese-1983-311-102	102	doigt	1303	FINGER	finger	L'ÊTRE HUMAIN
+Marchese-1983-311-103	103	dos	1291	BACK	back	L'ÊTRE HUMAIN
+Marchese-1983-311-104	104	épaule	1482	SHOULDER	shoulder	L'ÊTRE HUMAIN
+Marchese-1983-311-105	105	foie	1224	LIVER	liver	L'ÊTRE HUMAIN
+Marchese-1983-311-106	106	front	123	FOREHEAD	forehead	L'ÊTRE HUMAIN
+Marchese-1983-311-107	107	genou	1371	KNEE	knee	L'ÊTRE HUMAIN
+Marchese-1983-311-108	108	griffe	72	CLAW	claw	L'ÊTRE HUMAIN
+Marchese-1983-311-109	109	jambe	1297	LEG	leg	L'ÊTRE HUMAIN
+Marchese-1983-311-110	110	force			strength	L'ÊTRE HUMAIN
+Marchese-1983-311-111	111	langue	1205	TONGUE	language	L'ÊTRE HUMAIN
+Marchese-1983-311-112	112	menton	1510	CHIN	chin	L'ÊTRE HUMAIN
+Marchese-1983-311-113	113	nez	1221	NOSE	nose	L'ÊTRE HUMAIN
+Marchese-1983-311-114	114	nombril	1838	NAVEL	belly button	L'ÊTRE HUMAIN
+Marchese-1983-311-115	115	oeil	1248	EYE	eye	L'ÊTRE HUMAIN
```

Comment:
I don't know why the original source gives "o e i l", as it would be types "œ i l". Can this somehow be checked with the original?

Reply:
This one is reflected according to the original too and it can also be checked against other similar glosses in concepticon.

Reply:
Typesetters write oe traditionally as one character, but if our digital sources don't do that, we need to stick to them, as they should be reflected as they are. Just wanted to be sure, as one can easily confuse an œ with an oe.